src/Text/Pandoc/Class/PandocMonad.hs

```diff
@@ -586,6 +587,23 @@ readDataFile fname = do
             then readFileStrict (userDir </> fname)
             else readDefaultDataFile fname
 
+-- | Read metadata file from the working directory or, if not found there, from
+-- the metadata subdirectory of the user data directory.
+readMetadataFile :: PandocMonad m => FilePath -> m B.ByteString
+readMetadataFile fname = do
+  existsInWorkingDir <- fileExists fname
+  if existsInWorkingDir then readFileStrict fname
+  else do
+    dataDir <- getUserDataDir
```

Comment:
The style of the rest of the pandoc code is:
```
if foo
   then bar
   else baz
```
Better to keep to a consistent style.

Reply:
Changed in fixup.